### PR TITLE
Removes dorms toolboxes, tool storage insuls and multitools.

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
Gets rid of the dorms toolboxes, the tool storage multitools and replaces the remaining (meta, delta, kilo) tool storage insuls with budgets.
Added two multitools to the youtool premium section at 450 cr each.
Added a razor to kilostation dorms to fill the toolbox void.

## Why It's Good For The Game
Less freebies, more economy.

## Changelog
:cl: Skoglol
balance: Meta, Delta and Kilo tool storage insuls replaced with budgets.
balance: All stations have had their dorms toolboxes removed.
balance: Tool storage multitools removed, two added to the youtool premium section at 450cr each.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
why are multitools made engineer exclusive content

**Comment by ishitbyabullet** [comment]
Only an engineer can truly comprehend the multitool, Fikou.

You should probably move the toolboxes to the tool storage room unless there already is enough (No, only one electrical toolbox isn't enough) since those are iconic items or add in a vending machine. Would feel weird if getting them required the act of raiding Engineering each round.

**Comment by trollbreeder** [comment]
this just makes the assistants break into engineering instead of plundering tool storage
You can't control the tide kriskog, the tide controls you

**Comment by kriskog** [comment]
If you want your gamer gear, you should have to seek out the source of the gamer gear. No, I dont think multitools or insuls should be free in tool storage.

**Comment by ishitbyabullet** [comment]
Apparently you can print toolboxes in cargo.

Nevermind, assistants. You don't have to raid engineering for the toolbox. Only cargo now.

**Comment by Fikou** [comment]
this will just make more greys invade cargo by unanchoring the orm and printing 20 multitools

**Comment by Shadowflame909** [comment]
Please cease kriskog

Think of the shocked door power creep

Think of the lazy head of staff members, or security/head of staff in danger.

You just removed cloning madlad. Traitors can buy this stuff for 3 TC. Don't let there villainy be so unstoppable.

Tldr: if engineering does a shitty roundstart delam like they do 4/10 rounds. The antags are in for a good time

**Comment by kriskog** [comment]
Good point Fikou, why can we unanchor the ORM anyway? Someone should fix that.

**Comment by Dawson1917** [comment]
Why would you need a full set of tools unless you're 1) moonlighting engineering's job or 2) just using them to break into places as a non-antag? People with real jobs can buy tools they need from the vendor.

**Comment by Fikou** [comment]
> Why would you need a full set of tools unless you're 1) moonlighting engineering's job or 2) just using them to break into places as a non-antag? People with real jobs can buy tools they need from the vendor.

so you can do fun autism projects 
without that assistant is just bar rp or greytiding/validhunting

**Comment by MMMiracles** [comment]
sounds like you should play a job other than assistant if that's all you can think to do

**Comment by trollbreeder** [comment]
> sounds like you should play a job other than assistant if that's all you can think to do

Challenge: Name everything you think to do as an assistant. Going to the hop for a job, stealing 50% of engineering's job, tiding and validhunting do not count. Good luck

**Comment by Bawhoppen** [comment]
you can always assist people as assistant, since at their core they're meant to be a learning and/or low-responsibility role

**Comment by Shadowflame909** [comment]
Why everything cost money

**Comment by necromanceranne** [comment]
Jokes on you guys I just cut doors into walls.

**Comment by AdamElTablawy** [comment]
krisog at it again

at least 4dplanner made good contributions

**Comment by MCterra10** [comment]
Lol time to rush autolathe every round even more than we already do

**Comment by MMMiracles** [comment]
help, grug want free tool but code take way, grug no want crew interaction. grug only want grunt and smash for yellow glove. grug hate CHANGE

**Comment by Shadowflame909** [comment]
Grug makes a very good point here. /TG/ crewmembers are self centered and will call security on you for bothering them too much.

Leading to the "do it yourself" method which has lead to public mining and three RnD consoles.

Grug has also mentioned the game balance this efects with a lack of easy access to normal crew tools.

But what do I know. I'm just a botany main who ends up the most involved with economy changes.

I could barely afford the seeds. Now I have to afford the tools as well?

Give me a break.

**Comment by necromanceranne** [comment]
Botany consistently becomes more and more disfunctional with each of these changes just because within the average round you just can't do more elaborate botany stuff consistently without a chem dispenser. The major issue being that botany is a chem hungry role due to a complete rng nature of whether mutagen works and the constant need to get more saltpetre and ammonia. Nobody upgrades botany regularly because their machines are at the back and people consistently forget they need upgrades. And chemistry cannot reasonably output enough mutagen for them consistently while doing their own thing. And botany has to halt their production of anything else to make unstable mutagen, assuming they can reasonably do so with what mutagen they can get.

If they can't get mutagen, it's a dead role. No two ways about that. They just don't progress. 

Not the pr to discuss botany problems but it is one to discuss how fucking obscenely unrealistic interdepartmental co-operation being forced actually is. 

Good change for Manuel though. People might care enough to work together.

**Comment by MMMiracles** [comment]
Botany can mutate plants with mutagen, you know that right? Radium and uranium also work for mutations, but require 10u instead of 5. I know radium can come from at least glowshrooms when you grind them, and at that point you can easily make a plant that produces mutagen for you.

Seeds can probably have their prices looked at but forreal just fuckin' ask cargo to print you a set of tools for FREE it isn't that hard.

**Comment by necromanceranne** [comment]
You need to hack the vendors to get glowshrooms.

**Comment by wesoda25** [comment]
Botany isn’t dependent upon strange seeds thats not an argument.

However none of this stuff even poses an issue, seems unwarranted tbh

**Comment by Shadowflame909** [comment]
Botany is dependent on hacking the OTHER vendor. The one with chemicals that save plants that get too old. Or if you went afk/DC'd from plant  baby sitting simulator.

All I'm saying is since botany is basically the only job economy has an effect on. Since it's just free money to all the other jobs. You bump it up a tier.

**Comment by ATH1909** [comment]
> Botany isn’t dependent upon strange seeds/glowshrooms/whatever thats not an argument.
> 
> However none of this stuff even poses an issue, seems unwarranted tbh

bro do you even play botanist

And if you do, are you actually any good at it? Do you even gene mod your plants?

**Comment by HomocidlePancakes** [comment]
Dumb change. Fun thread.

**Review by Jalleo — APPROVED** [review]
Needed change. Makes tools rarer so they actually hold value to the crew members.

**Comment by AWalton** [comment]
Oh boy this sounds great as a botanist. Just yesterday I had a cargo tech plain refuse to let me use the autolathe to print stock parts even for a chem dispenser, when science and engi had no-one in the lobby to let me in to print better parts because they were autisming deep in their departments. 

Now I've got to give that same ass the power to say no to even giving me tools as well? Forced interdepartmental cooperation is a pile of shit when it's absolutely required to do the basics of your job.

**Comment by trollbreeder** [comment]
bikeshed alert

**Comment by ATH1909** [comment]
> Oh boy this sounds great as a botanist. Just yesterday I had a cargo tech plain refuse to let me use the autolathe to print stock parts even for a chem dispenser, when science and engi had no-one in the lobby to let me in to print better parts because they were autisming deep in their departments.
> 
> Now I've got to give that same ass the power to say no to even giving me tools as well? Forced interdepartmental cooperation is a pile of shit when it's absolutely required to do the basics of your job.

Pro-tip: while you're in tech storage for the chem dispenser board, pick up a battery. Go back to Botany and screwdriver-> crowbar a plant tray and an AI holopad, and that should give you the stock parts you need to make a dispenser (and a convenient machine frame to put them in).

**Comment by Tharcoonvagh** [comment]
nice

**Comment by AWalton** [comment]
> Pro-tip: while you're in tech storage for the chem dispenser board, pick up a battery. Go back to Botany and screwdriver-> crowbar a plant tray and an AI holopad, and that should give you the stock parts you need to make a dispenser (and a convenient machine frame to put them in).
That's a handy trick, but being forced to dismantle your own plant trays because lack of interdepartmental cooperation prevents early game job progression is bad gameplay. I can't think of any other job that requires it so badly for early game job activity.

**Comment by kriskog** [comment]
Requiring a chem dispenser in botany is bad gameplay.

**Comment by trollbreeder** [comment]
> Requiring a chem dispenser in botany is bad gameplay.

We have a solution, the botanical dispenser from the seed vault

**Comment by kriskog** [comment]
That's not a solution.

**Comment by trollbreeder** [comment]
> That's not a solution.

Hydroponics is a flawed department and the dispenser is a temporary solution before we can remake plant babysitting until gold trays then make the stupidest shit ever simulator into a better experience. For now, let them have their multitool and screwdriver.

**Comment by kriskog** [comment]
They can buy them. Or ask someone to make some, they are available in several other places. Lets not even pretend they usually get these first anyway, as there is a million assistants rushing tool storage every round.

Stop making this about botany when that is an edge case at best.

**Comment by Shadowflame909** [comment]
It was always about botany Kriskog. Because they're the only job actually effected by economy.

If security had to buy security jumpsuits and stun batons and flashbangs from that grey vendor they have. They'd be in a similar predicament.

Its one line change time

**Comment by dee-dubya** [comment]
Huh, that's not a bad idea actually. I wanted to kill tiding by randomizing the door wires, but killing tiding by taking away their free tools is actually a hell of a lot smarter.

Also, tiding cargo for the autolathe will be a lot harder without a wrench to take out meta's ORM.

**Comment by cacogen** [comment]
money's still feeling pretty worthless unless you're an assistant and they're who this affects

**Comment by kriskog** [comment]
Money will feel worthless until we add demand

**Comment by BadSS13Player** [comment]
> Money will feel worthless until we add enough demand

Sure, but this is a terrible way to add demand.

**Comment by GuyonBroadway** [comment]
The problem with interdepartmental cooperation is that on tg it usually works out as "in order for one department to do more of it's job, another department has to use thier resources to let them."

The problem is twofold, the other department seldom see's benefit from the first and if the second guy refuses the first is stuck unless they break in. Inter-departmental co-operation quickly becomes something approaching a banbait case.

**Comment by Ziiro** [comment]
No one is ever competent enough to make inter-departmental cooperation work. Or, the competent people are busy being antags and have a vested interest in not cooperating with you. The result of most "inter-departmental cooperation" goals and PRs simply end up making it more annoying to do anything because - guess what - no matter what your goal is, you're going to have to solve **every step** of your own problem yourself.

Are you science? Mining died, so no materials.
Are you mining? Robotics/Science spent points on robotics and didn't do upgrades OR mining tech.
Are you medical? Chemistry didn't do their job so you don't have any meds. R&D didn't grab any surgery upgrades, either.
Are you anything except engineering? Turns out, no engineer setup the SM (or wired the solars, even)
Are you security? Greytiders are causing more problems than actual antags.
Are you command? Delegating isn't working, because nobody listens.
Are you supply? Nobody is bringing in any of the bounties.

**Comment by ExcessiveUseOfCobblestone** [comment]
sometimes your rounds are not optimal and there's literally nothing you could do within the time frame of the round to make it optimal.

**Comment by ExcessiveUseOfCobblestone** [comment]
also remote signalers are multitools with a screwdriver btw and I think those are still fairly accessible.

**Comment by GuyonBroadway** [comment]
Gating a job behind what is tantamount to RNG is bad design.

**Comment by ExcessiveUseOfCobblestone** [comment]
which job are you referring to?

**Comment by Shadowflame909** [comment]
Wire  cutters can't restore bolted doors.

Only multitools can.

**Comment by GuyonBroadway** [comment]
Botany, oldsci, mining

Take your pick. Having job progress gated behind one coinflip dependant on someone else is shit design.

**Comment by JMoldy** [comment]
> Money will feel worthless until we add enough demand

Reworded: The entire game is going to be reworked to make one mechanic nobody who plays the game wants to have, works.

**Comment by Borisvanmemes** [comment]
I'd personally be fine with this if assistants actually got paid more, but I highly doubt that'll ever happen.

**Comment by BadSS13Player** [comment]
What? Why?
